### PR TITLE
SW-4389 Show empty batches filter option in "batches by seedlings" view

### PR DESCRIPTION
**Pull request by nickgraz**
- Add filter tooltip and popover to `InventorySeedlingsTable` with 'Show Empty Batches' boolean filter option
- Implement boolean filter type into `FilterGroup`
- Render filtered batches instead of searched batches in `InventorySeedlingsTable`.

### CODE REVIEW

File: src/components/common/FilterGroup/index.tsx [hunk 1]
```diff
@@ -32,20 +33,22 @@ const useStyles = makeStyles((theme: Theme) => ({
...
-    | 'hidden';
+    | 'hidden'
+    | 'boolean';
 };
 
 export type FilterGroupProps = {
   initialFilters: Record<string, SearchNodePayload>;
   fields: FilterField[];
-  values: FieldValuesPayload;
+  values?: FieldValuesPayload;
```
Comment (nickgraz) [inline src/components/common/FilterGroup/index.tsx:51]: This was only being used by `multiple_select` and `single_select` so I made it optional.

File: src/components/common/FilterGroup/index.tsx [hunk 2]
```diff
@@ -148,9 +153,18 @@ export default function FilterGroup(props: FilterGroupProps): JSX.Element {
                 payloads={filters[f.name]?.children ?? []}
               />
             )}
+            {f.type === 'boolean' && (
```
Comment (nickgraz) [inline src/components/common/FilterGroup/index.tsx:156]: Let me know if you'd rather I call this type `'checkbox'` and the component `FilterCheckbox`

Comment (charlie-tf) [reply src/components/common/FilterGroup/index.tsx:156]: `boolean` works for me

**Review by charlie-tf — COMMENTED** [review]
So Constanza and I have been putting the Inventory updates work in `src/components/InventoryV2`. I believe that `Search.tsx`, `FilterUtils.ts`, and `InventoryFiltersPopover.tsx` have been copied over to that directory, but are otherwise unchanged. How would it be to move your changes over there? Your `view/InventorySeedlingsTable.tsx` changes would have to be ported over there, too, and I'm not quite sure what routing changes might need to be effected to get this to work right under the Nursery Updates switch.

**Comment by charlie-tf** [comment]
Besides that, what you have here looks good!

**Comment by nickgraz** [comment]
Ah my bad, I can move it over to the V2 area

**Review by charlie-tf — APPROVED** [review]
Thanks for the rearrangement. Looks good and tested working in Vercel.

**Pull request merged**